tools/php-cs-fixer/composer.json

```
@@ -0,0 +1,6 @@
+{
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "^3.59",
+        "ibexa/code-style": "^1.0"
```

```suggestion
        "ibexa/code-style": "~1.2.0"
```
or
```suggestion
        "ibexa/code-style": "~2.0.0"
```

We're using minor versions for code style sets, and prefer to use patch versions.